brainscore_language/benchmarks/pereira2018/test.py

```diff
@@ -3,40 +3,47 @@
 import pytest
 from numpy.random import RandomState
 from pytest import approx
+from typing import Callable, Union, List
 
 from brainio.assemblies import NeuroidAssembly
 from brainscore_language import ArtificialSubject, load_benchmark
 
 
 class TestBenchmark:
     class DummyModel(ArtificialSubject):
-        def __init__(self, neural_activity):
-            self.neural_activity = neural_activity
+        def __init__(self, activity_for_text: Callable[[Union[str, List[str]]], NeuroidAssembly]):
+            self.activity_for_text = activity_for_text
 
         def digest_text(self, stimuli):
-            assert len(stimuli) == len(self.neural_activity['presentation'])
-            self.neural_activity['stimulus'] = 'presentation', stimuli  # copy over
-            return {'neural': self.neural_activity}
+            neural_activity = self.activity_for_text(stimuli)
+            return {'neural': neural_activity}
 
         def start_neural_recording(self, recording_target: ArtificialSubject.RecordingTarget,
                                    recording_type: ArtificialSubject.RecordingType):
             assert recording_target == ArtificialSubject.RecordingTarget.language_system
             assert recording_type == ArtificialSubject.RecordingType.fMRI
 
     @pytest.mark.parametrize('experiment, expected_score', [
-        (243, approx(0.0017534 / .35378928, abs=0.001)),
+        (243, approx(0.00659811 / .35378928, abs=0.001)),
```

Comment:
I know this was here previously, but what are these "/ .35*". Is that raw-score divided by ceiling? If so, could you add a comment somewhere.

Reply:
yes exactly; thanks for pointing this out, will just put the score